.gitignore

```diff
@@ -4,4 +4,5 @@ packages/**/dist/
 node_modules/
 .editorconfig
 .eslintcache
+.idea/
```

Comment:
It's better to push `.idea` ignore to `~/.gitignore` since it's system specific.